buildSrc/src/main/kotlin/Platforms.kt

```diff
@@ -38,20 +43,28 @@ private val appleTargets = listOf(
     "tvosSimulatorArm64",
     "watchosArm32",
     "watchosArm64",
-    "watchosX86",
     "watchosX64",
-    "watchosSimulatorArm64"
+    "watchosSimulatorArm64",
+    "watchosDeviceArm64"
 )
 
 private val mingwTargets = listOf(
     "mingwX64"
 )
 
 private val linuxTargets = listOf(
-    "linuxX64"
+    "linuxX64",
+    "linuxArm64"
 )
 
-val nativeTargets = appleTargets + linuxTargets + mingwTargets
+private val androidTargets = listOf(
+    "androidNativeArm32",
+    "androidNativeArm64",
+    "androidNativeX64",
+    "androidNativeX86"
+)
+
+val nativeTargets = appleTargets + linuxTargets + mingwTargets + androidTargets
```

Comment:
minor: new kotlin target hierarchy API could be used instead: https://kotlinlang.org/docs/whatsnew1820.html#new-approach-to-source-set-hierarchy

Reply:
https://github.com/Kotlin/kotlinx-io/issues/147